src/sirhelpers.c

```diff
@@ -105,10 +105,10 @@ bool __sir_validupdatedata(const sir_update_config_data* data, const char* func,
     if (valid && _sir_bittest(data->fields, SIRU_SYSLOG_CAT))
         valid = _sir_validstrnofail(data->sl_category);
 
-    if (!valid) {
+    if (!valid) { // GCOVR_EXCL_START
```

Comment:
what's this aboot? it would be very easy to make a test to cover that unless the setter function won't let you set an empty string

Reply:
I tried and I couldn't get any coverage here consistently...  I'm not sure why